libbeat/metric/system/process/process.go

```diff
@@ -460,6 +460,35 @@ func (procStats *Stats) GetOne(pid int) (common.MapStr, error) {
 	return e, nil
 }
 
+func (procStats *Stats) getSingleProcess1(pid int, newProcs ProcsMap) (*Process, error) {
```

Comment:
Just curious, why not just add the return error on original function `getSingleProcess`? 😬

Reply:
@kaiyan-sheng , I have updated the description with more info on this, this PR was not meant to be reviewed, not sure why it was moved to inbox, I have set it back to in progress.